backend/app/routes/faq/updateFaq.js

```diff
@@ -1,3 +1,10 @@
+/* eslint-disable no-shadow */
+/* eslint-disable prefer-arrow-callback */
+/* eslint-disable prefer-destructuring */
+/* eslint-disable prefer-const */
+/* eslint-disable consistent-return */
+/* eslint-disable no-unused-vars */
+/* eslint-disable prettier/prettier */
```

Comment:
can you please remove these un-necessary changes?

Reply:
@Kajol-Kumari If i remove those 4 line while commiting the changes it askes to change the entire file.